ntbs-service/DataMigration/NotificationMapper.cs

```diff
@@ -512,6 +518,20 @@ private static ComorbidityDetails ExtractComorbidityDetails(MigrationDbNotificat
             return details;
         }
 
+        private ICollection<PreviousTbService> ExtractPreviousTbServices(IEnumerable<TreatmentEvent> notificationTransferEvents)
+        {
+            return notificationTransferEvents
+                .Where(e => e.TreatmentEventType is TreatmentEventType.TransferOut && e.EventDate.HasValue)
+                .Select(e => new PreviousTbService
+                {
+                    TbServiceCode = e.TbServiceCode,
+                    TransferDate = e.EventDate.Value,
+                    NotificationId = e.NotificationId,
```

Comment:
Do we actually have a notification ID to set at this point? I would have expected it only to exist later when the notification is actually saved.